Gruntfile.js

```diff
@@ -227,83 +192,58 @@ module.exports = function(grunt) {
                 src: 'build/test-coverage/lcov/**/*.info',
             }
         },
-        // file concatenation
-        concat: {
-            options: {
-                banner: cssFileHeader,
-            },
-            // css for the "page" template
-            pageCss: {
-                src: pageCssFiles,
-                dest: kbaseStaticDir + 'css/page_concat.css',
-                nonull: true,
-            },
-            // a single file containing all the KBase css!
-            allCss: {
-                src: allCssFiles,
-                dest: kbaseStaticDir + 'css/all_concat.css',
-                nonull: true,
-            }
-        },
 
         // check the source css files are not completely horrendous
+        // config to be updated for scss files
```

Comment:
This task doesn't do anything at the moment as I haven't yet configured the stylelint settings for scss files.